cdc/cdc/model/kv_gen.go

```diff
@@ -206,15 +212,24 @@ func (z *RawKVEntry) EncodeMsg(en *msgp.Writer) (err error) {
 		err = msgp.WrapError(err, "RegionID")
 		return
 	}
+	err = en.Append(0xaa, 0x6b, 0x65, 0x79, 0x73, 0x70, 0x61, 0x6e, 0x5f, 0x69, 0x64)
+	if err != nil {
+		return
+	}
+	err = en.WriteUint64(z.KeySpanID)
+	if err != nil {
+		err = msgp.WrapError(err, "KeySpanID")
+		return
+	}
 	return
 }
 
 // MarshalMsg implements msgp.Marshaler
 func (z *RawKVEntry) MarshalMsg(b []byte) (o []byte, err error) {
 	o = msgp.Require(b, z.Msgsize())
-	// map header, size 7
+	// map header, size 8
 	// string "op_type"
-	o = append(o, 0x87, 0xa7, 0x6f, 0x70, 0x5f, 0x74, 0x79, 0x70, 0x65)
+	o = append(o, 0x88, 0xa7, 0x6f, 0x70, 0x5f, 0x74, 0x79, 0x70, 0x65)
```

Comment:
Should this be same with `EncodeMsg` func?

Reply:
8 means that there are 8 fields, example https://msgpack.org/.

Reply:
Yes, I will modify.